pkg/execution/batch/redis.go

```diff
@@ -90,6 +91,8 @@ func (b redisBatchManager) Append(ctx context.Context, bi BatchItem, fn inngest.
 		return nil, fmt.Errorf("no batch config found for for function: %s", fn.Slug)
 	}
 
+	client := b.b.ShardedClient()
```

Comment:
Append is always sharded from now on. This means that new batches will be created on the sharded cluster. Previous batches will simply run into the configured or max timeout (60s) and execute. This way we don't need to specify a rollout policy, while still handling all batches in the system. After a minute past rolling out, all existing batches will have been processed and all new batches are sharded.